app/src/main/res/layout/activity_main.xml

```diff
@@ -37,17 +38,45 @@
                     android:background="@color/white"
                     android:elevation="@dimen/elevation_card"/>
 
-                <ProgressBar
-                    android:id="@+id/empty_list_item"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:elevation="@dimen/elevation_refresh_indicator"
+                <android.support.constraint.ConstraintLayout
+                    android:id="@+id/emptyListItem"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
                     android:visibility="gone"
-                    app:layout_constraintBottom_toTopOf="parent"
-                    app:layout_constraintEnd_toStartOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toTopOf="parent" />
-            </android.support.constraint.ConstraintLayout>
+                    app:layout_constraintTop_toTopOf="parent">
+
+                    <com.airbnb.lottie.LottieAnimationView
+                        android:id="@+id/emptyListItem1"
```

Comment:
An id that indicated what this is would be useful, e.g. `user_list_loading_item`, `user_list_loading_animation`, `user_list_loading_text`. These IDs are kind of the equivalent of `String string1; String string2; int num;`.